### ISSUE TITLE
influx database create cause 40x response when http-auth is enabled in influx

### ISSUE THREAD

**Issue by bckfnn** [micrometer-metrics/micrometer#389]
The auto-create database feature in InfluxMeterRegistry  does not send auth info, so every publish of event to influx cause an log entry with ERROR level in line 

```
2018-02-02 08:19:50,021 ERROR io.micrometer.influx.InfluxMeterRegistry:78 -   - unable to create database 'metric': {"error":"unable to parse authentication credentials"}
```

I would prefer that the database create feature 
- could be disabled
- would send the configured auth

**Comment by jkschneider** [comment]
Great feedback @bckfnn. Will do.

**Comment by jkschneider** [comment]
Fixed in d7ea9a38.
